src/views/AssignPickerModal.vue

```diff
@@ -71,14 +80,153 @@ export default defineComponent({
+      orders.map((order) => {
+        order.doclist.docs.map((item) => items.push(item))
+      });
+
+      items.map((item, index) => {
+        formData.append("facilityId_o_"+index, this.currentFacility.facilityId)
+        formData.append("shipmentMethodTypeId_o_"+index, item.shipmentMethodTypeId)
+        formData.append("itemStatusId_o_"+index, "PICKITEM_PENDING")
+        formData.append("shipGroupSeqId_o_"+index, item.shipGroupSeqId)
+        formData.append("orderId_o_"+index, item.orderId)
+        formData.append("orderItemSeqId_o_"+index, item.orderItemSeqId)
+        formData.append("productId_o_"+index, item.productId)
+        formData.append("quantity_o_"+index, item.itemQuantity)
+        formData.append("inventoryItemId_o_"+index, item.inventoryItemId)
+        formData.append("picked_o_"+index, item.itemQuantity)
+        formData.append("_rowSubmit_o_"+index, "Y")
+      })
+
+      // Adding all the pickers selected in FormData
+      // TODO: check if we need to only allow selection of 3 or less pickers as in the current fulfillment app we can only select 3 pickers
+      this.selectedPickers.map((picker) => formData.append("pickerIds", picker.id))
+
+      try {
+        resp = await UtilService.createPicklist(formData);
+        if (resp.status === 200 && !hasError(resp)) {
+          this.closeModal();
+          await this.store.dispatch('order/fetchOpenOrders')
+        } else {
+          showToast(translate('Failed to create picklist for orders'))
+        }
+      } catch (err) {
+        console.error(err)
+        showToast(translate('Something went wrong'))
+      }
+
+      emitter.emit("dismissLoader")
+    },
+    async fetchPickers() {
+      let inputFields = {}
+
+      if(this.queryString.length > 0) {
+        inputFields = {
+          firstName_value: this.queryString,
+          firstName_op: 'contains',
+          firstName_ic: 'Y',
+          firstName_grp: '1',
+          externalId_value: this.queryString,
+          externalId_op: 'contains',
+          externalId_ic: 'Y',
+          externalId_grp: '2',
+          lastName_value: this.queryString,
+          lastName_op: 'contains',
+          lastName_ic: 'Y',
+          lastName_grp: '3',
+          partyId_value: this.queryString,
+          partyId_op: 'contains',
+          partyId_ic: 'Y',
+          partyId_grp: '4',
+        }
+      }
+
+      const payload = {
+        inputFields: {
+          ...inputFields,
+          roleTypeIdTo: 'WAREHOUSE_PICKER'
+        },
+        viewSize: 50,
+        entityName: 'PartyRelationshipAndDetail',
+        noConditionFind: 'Y',
+        orderBy: "firstName ASC",
+        filterByDate: "Y",
+        distinct: "Y",
+        fieldList: ["firstName", "lastName", "partyId"]
+      }
+
+      try {
+        const resp = await UtilService.getAvailablePickers(payload);
+        if (resp.status === 200 && !hasError(resp) && resp.data.count > 0) {
```

Comment:
We should not show "Something went wrong" error when there are no pickers found. It should be empty state.

Reply:
Sir, we are not displaying it on the UI, we are just adding the error in console for debugging, also we already have empty state for the same.
I'll update the message to be more informative.